### DISCUSSION TITLE
Offline problem

### DISCUSSION THREAD

**Discussion by Lupu27** [TeamVanced/VancedManager#692] [General]
I have a problem with Youtube Vanced, every time I enter the application it tells me that I am offline, I reinstalled the application and it tells me the same 
![Screenshot_20210929-203332](https://user-images.githubusercontent.com/91627370/135319898-a8639e5e-1014-4bcc-b081-6b7f1fb27da0.jpg)

**Comment by sup2069** [comment]
Heres a possible fix from a dev: 

https://www.reddit.com/r/Vanced/comments/ozsdpb/fixing_vanced_not_being_able_to_connect_to_the/?utm_medium=android_app&utm_source=share
